easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -16,8 +16,9 @@ exts_defaultclass = 'PythonPackage'
 
 dependencies = [
     ('Python', '2.7.14'),
+    ('Tkinter', '%(pyver)s', versionsuffix),
     ('libpng', '1.6.32'),
-    ('freetype', '2.8.1'),
+    ('freetype', '2.8'),
```

Comment:
@schiotz Please stick to `('freetype', '2.8.1')` here, to avoid introducing conflicts.

Reply:
@boegel Also here, I had to change the freetype version from 2.8.1 to 2.8, since otherwise I got a conflict with Tkinter or one of its dependencies.  But I can certainly believe that my change causes conflicts on packages depending on matplotlib; is that what you see?

In which case I do not quite know how to solve this ... :-(